content/src/view.js

```diff
@@ -1600,6 +1604,19 @@ function updatePaneTitle(pane) {
       label = 'data';
     } else if (/^text\/html/.test(paneState.mimeType)) {
```

Comment:
Let's move this HTML case inside the "else" below - it looks like the only difference is `label = 'html'` vs `'code'`, so let's just put the html `if` around that one small difference.